R/plumber_functions.R

```diff
@@ -46,18 +48,20 @@ generate_from_gs = function(link,
     model_name = model_name,
     vocoder_name = vocoder_name
   )
-  # GET
-  response <- httr::GET(
+  # POST
+  response <- httr::POST(
     url = paste0(api_url, "/generate_from_gs"),
     query = body,
-    ...)
+    httr::write_disk(output))
 
-  # Originally from mario::mario_write_video()
-  httr::stop_for_status(response)
-  bin_data = httr::content(response)
-  output = tempfile(fileext = ".mp4")
-  writeBin(bin_data, output)
+  # Tell user whether POST request has been completed.
+  response_status <- httr::stop_for_status(response)
+  if (response_status$status_code == 200) {
+    message(paste0("Rendered video is available in your working directory as ", output))
```

Comment:
This part might have to change with a future PR. I think the thought was that it would return an object that a user could `GET` the status of to see if their video was complete or an estimated time of when their video would be complete. 

So they would post the request but then instead a object they could query to see how their video is doing in the queue would return. 

(But this all can be added step by step)